ParisBNF/abb/BNFabb105.xml

```diff
@@ -935,20 +935,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <history>
                         <origin>
                      <origDate notBefore="1482" notAfter="1483" cert="medium"/>The
-                     manuscript was commissioned <persName ref="PRS2635BelenSa" role="owner">Bǝlen Sagad</persName>,
+                     manuscript was commissioned by <persName ref="PRS2635BelenSa" role="owner">Bǝlen Sagad</persName>,
                        <foreign xml:lang="gez">ʾaqāḍan</foreign> of
-                       <placeName ref="LOC5575Saraye">Saruwe</placeName>. According to
-                       <bibl><ptr target="bm:ContiRossini1914Notice"/>
-                          <citedRange unit="page">47</citedRange>
-                       </bibl>
+                       <placeName ref="LOC5575Saraye">Saruwe</placeName>. According to Conti Rossini,
                        <persName ref="PRS13319TanseaKrestos" role="scribe" cert="low">Tanśǝʾa Krǝstos</persName>
-                        mentioned in supplications on some folia (e.g. <ref target="#139ra #143ra"></ref>) might have been the scribe.
+                        mentioned in supplications on some folia (e.g. <ref target="#139ra #143ra"></ref>) might have
+                        been the scribe who copied the text.
                      </origin>
                      <provenance>The manuscript was bought by
                        <persName ref="PRS1071dAbbadi" role="owner">Antoine d'Abbadie</persName>
                        sometime between <date>1840</date> and <date>1848</date>.
                      </provenance>
-                     <acquisition>The manuscript entered the Bibliothèque nationale in
+                     <acquisition>The manuscript entered<placeName ref="INS0303BNF"/> in
```

Comment:
you need a space before the tag, and no space inside of the tag before the /

Reply:
ok